tools/tpm2_nvlist.c

```diff
@@ -42,9 +42,15 @@
 #include "options.h"
 
 static void print_nv_public(TPM2B_NV_PUBLIC *nv_public) {
+
+    char *attrs = tpm2_nv_util_attrtostr(nv_public->t.nvPublic.attributes);
+    if (!attrs) {
+        LOG_ERR("Could not convert attributes to string form");
+    }
+
     printf("  {\n");
     printf("\tHash algorithm(nameAlg):%d\n ", nv_public->t.nvPublic.nameAlg);
-    printf("\tThe Index attributes(attributes):0x%x\n ",
+    printf("\tattributes: %s(0x%x)\n ", attrs,
             nv_public->t.nvPublic.attributes.val);
```

Comment:
Oh I spotted a bug on this  yesterday, the atributes.val needs a network to host byte order conversion... ill add a patch to this PR and file a bug.